scripts/windows/build/rocksdb-win-arm32.ps1

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+function Get-Rocksdb-WinArm32
+{
+    $ErrorActionPreference = 'Continue'
+
+    $vcpkgroot = "$env:HOMEDRIVE\vcpkg1"
+
+    if (!(Test-Path -Path $vcpkgroot))
+    {
+        Write-Host "git clone vcpkg"
+        git clone https://github.com/Microsoft/vcpkg $vcpkgroot
+        if ($LastExitCode)
+        {
+            Throw "Failed to clone vcpkg repo with exit code $LastExitCode"
+        }
+    }
+
+    # checkout the specific commit that matches the rocksdbsharp being used
+    # 4d224838df2da68def51540e328b8b692ae5607f maps to 5.18.3
```

Comment:
If ARM32 is going to use 5.18.3 then everything else (Windows x64 and all Linux) should use it as well. We shouldn't have different versions for different arches.